ecs/run-task/main.go

```diff
@@ -21,10 +25,36 @@ func main() {
 
 	ecsClient := ecs.New(session, conf)
 
+	taskOverride, overridesErr := resolveTaskOverride(taskOverrideJSON)
+	common.FatalOnError(overridesErr)
+
 	_, err := ecsClient.RunTask(&ecs.RunTaskInput{
 		TaskDefinition: taskDefinition,
 		Cluster:        cluster,
 		Count:          aws.Int64(1),
+		Overrides:      taskOverride,
 	})
+
 	common.FatalOnError(err)
 }
+
+func resolveTaskOverride(taskOverridesJSON *string) (*ecs.TaskOverride, error) {
+
+	if *taskOverridesJSON == "" {
```

Comment:
instead of passing the pointer to the function, de-reference it in the main and pass it as `string`. From the main scope we know it's safe to de-reference it, but from the function itself it's not obvious so that line looks unsafe.